tests.sh

```diff
@@ -234,6 +282,66 @@ build_python_cpp() {
   cd ..
 }
 
+build_python27_cpp() {
+  internal_build_cpp
+  export LD_LIBRARY_PATH=../src/.libs # for Linux
+  export DYLD_LIBRARY_PATH=../src/.libs # for OS X
+  cd python
+    envlist=py27-cpp
```

Comment:
why is this indented? Same comment below

Reply:
Removed indent.